### ISSUE TITLE
Startup Error

### ISSUE THREAD

**Issue by Sewlell** [karaoke-dev/karaoke#2141]
```
2023-10-01 04:43:06 [verbose]: ----------------------------------------------------------
2023-10-01 04:43:06 [verbose]: runtime Log for user name (LogLevel: Verbose)
2023-10-01 04:43:06 [verbose]: Running osu 2023.924.1.0 on .NET 6.0.22
2023-10-01 04:43:06 [verbose]: Environment: Windows (Microsoft Windows NT 10.0.22621.0), 8 cores
2023-10-01 04:43:06 [verbose]: ----------------------------------------------------------
2023-10-01 04:43:06 [verbose]: 🖼️ Configuration renderer choice: Automatic
2023-10-01 04:43:06 [verbose]: 🖼️ Renderer fallback order: [ Direct3D 11, OpenGL (Legacy), OpenGL, Vulkan ]
2023-10-01 04:43:06 [verbose]: 🖼️ Initialising "Veldrid" renderer with "Direct3D11" surface
2023-10-01 04:43:06 [verbose]: Updated display mode to desktop resolution: 1920x1080@60, SDL_PIXELFORMAT_RGB888
2023-10-01 04:43:07 [verbose]: Direct3D 11 Initialized
2023-10-01 04:43:07 [verbose]: Direct3D 11 Feature Level:           11.0
2023-10-01 04:43:07 [verbose]: Direct3D 11 Adapter:                 NVIDIA GeForce MX130
2023-10-01 04:43:07 [verbose]: Direct3D 11 Dedicated Video Memory:  2008 MB
2023-10-01 04:43:07 [verbose]: Direct3D 11 Dedicated System Memory: 0 MB
2023-10-01 04:43:07 [verbose]: Direct3D 11 Shared System Memory:    6045 MB
2023-10-01 04:43:07 [verbose]: UseStructuredBuffers: True
2023-10-01 04:43:07 [verbose]: 🖼️ Renderer initialised!
2023-10-01 04:43:07 [verbose]: Host execution state changed to Running
2023-10-01 04:43:07 [verbose]: Execution mode changed to MultiThreaded
2023-10-01 04:43:07 [verbose]: Beginning realm file store cleanup
2023-10-01 04:43:07 [verbose]: Finished realm file store cleanup (0 of 20809 deleted)
2023-10-01 04:43:07 [verbose]: 🔈 BASS initialised
2023-10-01 04:43:07 [verbose]: BASS version:           2.4.16.7
2023-10-01 04:43:07 [verbose]: BASS FX version:        2.4.12.6
2023-10-01 04:43:07 [verbose]: BASS MIX version:       2.4.11.1
2023-10-01 04:43:07 [verbose]: Device:                 Default
2023-10-01 04:43:07 [verbose]: Driver:
2023-10-01 04:43:07 [verbose]: Update period:          5 ms
2023-10-01 04:43:07 [verbose]: Device buffer length:   10 ms
2023-10-01 04:43:07 [verbose]: Playback buffer length: 100 ms
2023-10-01 04:43:07 [verbose]: 🔈 BASS initialised
2023-10-01 04:43:07 [verbose]: BASS version:           2.4.16.7
2023-10-01 04:43:07 [verbose]: BASS FX version:        2.4.12.6
2023-10-01 04:43:07 [verbose]: BASS MIX version:       2.4.11.1
2023-10-01 04:43:07 [verbose]: Device:                 Default
2023-10-01 04:43:07 [verbose]: Driver:
2023-10-01 04:43:07 [verbose]: Update period:          5 ms
2023-10-01 04:43:07 [verbose]: Device buffer length:   10 ms
2023-10-01 04:43:07 [verbose]: Playback buffer length: 100 ms
2023-10-01 04:43:08 [verbose]: Opened realm "C:\Users\user name\AppData\Roaming\osu\client.realm" at version 34
2023-10-01 04:43:08 [verbose]: 🖍️ Shader Texture2D/Texture loaded from cache!
2023-10-01 04:43:08 [verbose]: 🖍️ Shader Texture2D/CircularProgress loaded from cache!
2023-10-01 04:43:08 [verbose]: Updated display mode to fullscreen resolution: 1920x1080@60, SDL_PIXELFORMAT_RGB888
2023-10-01 04:43:08 [verbose]: Loading FPSCounter...
2023-10-01 04:43:08 [error]: An unhandled error has occurred.
2023-10-01 04:43:08 [error]: System.Globalization.CultureNotFoundException: Culture is not supported. (Parameter 'name')
2023-10-01 04:43:08 [error]: 中文（简体） is an invalid culture identifier.
2023-10-01 04:43:08 [error]: at System.Globalization.CultureInfo..ctor(String name, Boolean useUserOverride)
2023-10-01 04:43:08 [error]: at System.Globalization.CultureInfo..ctor(String name)
2023-10-01 04:43:08 [error]: at osu.Game.Rulesets.Karaoke.Utils.CultureInfoUtils.CreateLoadCultureInfoByCode(String code)
2023-10-01 04:43:08 [error]: at osu.Game.Rulesets.Karaoke.Bindables.BindableCultureInfo.Parse(Object input)
2023-10-01 04:43:08 [error]: at osu.Game.Rulesets.Configuration.RulesetConfigManager`1.AddBindable[TBindable](TLookup lookup, Bindable`1 bindable)
2023-10-01 04:43:08 [error]: at osu.Game.Rulesets.Karaoke.Configuration.KaraokeRulesetConfigManager.AddBindable[TBindable](KaraokeRulesetSetting lookup, Bindable`1 bindable)
2023-10-01 04:43:08 [error]: at osu.Framework.Configuration.ConfigManager`1.set[TValue](TLookup lookup, TValue value)
2023-10-01 04:43:08 [error]: at osu.Framework.Configuration.ConfigManager`1.SetDefault[TValue](TLookup lookup, TValue value)
2023-10-01 04:43:08 [error]: at osu.Game.Rulesets.Karaoke.Configuration.KaraokeRulesetConfigManager.InitialiseDefaults()
2023-10-01 04:43:08 [error]: at osu.Game.Rulesets.Configuration.RulesetConfigManager`1..ctor(SettingsStore store, RulesetInfo ruleset, Nullable`1 variant)
2023-10-01 04:43:08 [error]: at osu.Game.Rulesets.Karaoke.Configuration.KaraokeRulesetConfigManager..ctor(SettingsStore settings, RulesetInfo ruleset, Nullable`1 variant)
2023-10-01 04:43:08 [error]: at osu.Game.Rulesets.Karaoke.KaraokeRuleset.CreateConfig(SettingsStore settings)
2023-10-01 04:43:08 [error]: at osu.Game.Rulesets.RulesetConfigCache.LoadComplete()
2023-10-01 04:43:08 [error]: at osu.Framework.Graphics.Drawable.loadComplete()
2023-10-01 04:43:08 [error]: at osu.Framework.Graphics.Drawable.UpdateSubTree()
2023-10-01 04:43:08 [error]: at osu.Framework.Graphics.Containers.CompositeDrawable.UpdateSubTree()
2023-10-01 04:43:08 [error]: at osu.Framework.Graphics.Containers.CompositeDrawable.UpdateSubTree()
2023-10-01 04:43:08 [error]: at osu.Framework.Graphics.Containers.CompositeDrawable.UpdateSubTree()
2023-10-01 04:43:08 [error]: at osu.Framework.Graphics.Containers.CompositeDrawable.UpdateSubTree()
2023-10-01 04:43:08 [error]: at osu.Framework.Graphics.Containers.CompositeDrawable.UpdateSubTree()
2023-10-01 04:43:08 [error]: at osu.Framework.Graphics.Containers.CompositeDrawable.UpdateSubTree()
2023-10-01 04:43:08 [error]: at osu.Framework.Platform.GameHost.UpdateFrame()
2023-10-01 04:43:08 [error]: at osu.Framework.Threading.GameThread.processFrame()
2023-10-01 04:43:08 [verbose]: Unhandled exception has been allowed with 0 more allowable exceptions .
2023-10-01 04:43:08 [verbose]: Loaded FPSCounter!
2023-10-01 04:43:08 [verbose]: Loading VersionManager...
2023-10-01 04:43:09 [verbose]: Loaded VersionManager!
2023-10-01 04:43:09 [verbose]: Loading OsuLogo...
2023-10-01 04:43:09 [verbose]: 🖍️ Shader Texture2D/TriangleBorder loaded from cache!
2023-10-01 04:43:09 [verbose]: Loaded OsuLogo!
2023-10-01 04:43:09 [verbose]: Loading Toolbar...
2023-10-01 04:43:09 [verbose]: 📺 OsuScreenStack#502(depth:1) loading Loader#532
2023-10-01 04:43:09 [verbose]: 📺 OsuScreenStack#502(depth:1) entered Loader#532
2023-10-01 04:43:09 [verbose]: 🖍️ Shader Texture2D/Blur loaded from cache!
2023-10-01 04:43:09 [verbose]: 🖍️ Shader CursorTrail/Texture loaded from cache!
2023-10-01 04:43:09 [verbose]: 🖍️ Shader Texture3D/Texture loaded from cache!
2023-10-01 04:43:09 [verbose]: Loaded Toolbar!
2023-10-01 04:43:09 [verbose]: Loading VolumeOverlay...
2023-10-01 04:43:09 [verbose]: Loaded VolumeOverlay!
2023-10-01 04:43:09 [verbose]: Loading OnScreenDisplay...
2023-10-01 04:43:09 [verbose]: Loaded OnScreenDisplay!
2023-10-01 04:43:09 [verbose]: Loading NotificationOverlay...
2023-10-01 04:43:09 [verbose]: Loaded NotificationOverlay!
2023-10-01 04:43:09 [verbose]: Loading LegacyImportManager...
2023-10-01 04:43:09 [verbose]: Loaded LegacyImportManager!
2023-10-01 04:43:09 [verbose]: Loading ScreenshotManager...
2023-10-01 04:43:09 [verbose]: Loaded ScreenshotManager!
2023-10-01 04:43:09 [verbose]: Loading SquirrelUpdateManager...
2023-10-01 04:43:09 [verbose]: Loaded SquirrelUpdateManager!
2023-10-01 04:43:09 [verbose]: Loading FirstRunSetupOverlay...
2023-10-01 04:43:09 [verbose]: Loaded FirstRunSetupOverlay!
2023-10-01 04:43:09 [verbose]: Loading ManageCollectionsDialog...
2023-10-01 04:43:09 [verbose]: Loaded ManageCollectionsDialog!
2023-10-01 04:43:09 [verbose]: Loading BeatmapListingOverlay...
2023-10-01 04:43:09 [verbose]: Loaded BeatmapListingOverlay!
2023-10-01 04:43:09 [verbose]: Loading DashboardOverlay...
2023-10-01 04:43:09 [verbose]: Loaded DashboardOverlay!
2023-10-01 04:43:09 [verbose]: Loading NewsOverlay...
2023-10-01 04:43:09 [verbose]: Loaded NewsOverlay!
2023-10-01 04:43:09 [verbose]: Loading RankingsOverlay...
2023-10-01 04:43:09 [verbose]: Loaded RankingsOverlay!
2023-10-01 04:43:09 [verbose]: Loading ChannelManager...
2023-10-01 04:43:09 [verbose]: Loaded ChannelManager!
2023-10-01 04:43:09 [verbose]: Loading ChatOverlay...
2023-10-01 04:43:09 [verbose]: Loaded ChatOverlay!
2023-10-01 04:43:09 [verbose]: Loading MessageNotifier...
2023-10-01 04:43:09 [verbose]: Loaded MessageNotifier!
2023-10-01 04:43:09 [verbose]: Loading SettingsOverlay...
2023-10-01 04:43:09 [verbose]: Loaded SettingsOverlay!
2023-10-01 04:43:09 [verbose]: Loading ChangelogOverlay...
2023-10-01 04:43:09 [verbose]: Loaded ChangelogOverlay!
2023-10-01 04:43:09 [verbose]: Loading UserProfileOverlay...
2023-10-01 04:43:09 [verbose]: Loaded UserProfileOverlay!
2023-10-01 04:43:09 [verbose]: Loading BeatmapSetOverlay...
2023-10-01 04:43:10 [verbose]: Loaded BeatmapSetOverlay!
2023-10-01 04:43:10 [verbose]: Loading WikiOverlay...
2023-10-01 04:43:10 [verbose]: Loaded WikiOverlay!
2023-10-01 04:43:10 [verbose]: Loading SkinEditorOverlay...
2023-10-01 04:43:10 [verbose]: Loaded SkinEditorOverlay!
2023-10-01 04:43:10 [verbose]: Loading LoginOverlay...
2023-10-01 04:43:10 [verbose]: Loaded LoginOverlay!
2023-10-01 04:43:10 [verbose]: Loading NowPlayingOverlay...
2023-10-01 04:43:10 [verbose]: Loaded NowPlayingOverlay!
2023-10-01 04:43:10 [verbose]: Loading AccountCreationOverlay...
2023-10-01 04:43:10 [verbose]: 📺 ScreenStack#263(depth:1) scheduling push ScreenWelcome#435
2023-10-01 04:43:10 [verbose]: Loaded AccountCreationOverlay!
2023-10-01 04:43:10 [verbose]: Loading DialogOverlay...
2023-10-01 04:43:10 [verbose]: Loaded DialogOverlay!
2023-10-01 04:43:10 [verbose]: Loading HighPerformanceSession...
2023-10-01 04:43:10 [verbose]: Loaded HighPerformanceSession!
2023-10-01 04:43:10 [verbose]: Loading BackgroundDataStoreProcessor...
2023-10-01 04:43:10 [verbose]: Beginning background data store processing..
2023-10-01 04:43:10 [verbose]: Querying for beatmap sets to reprocess...
2023-10-01 04:43:10 [verbose]: Loaded BackgroundDataStoreProcessor!
2023-10-01 04:43:10 [verbose]: Found 0 beatmap sets which require reprocessing.
2023-10-01 04:43:10 [verbose]: Querying for scores to reprocess...
2023-10-01 04:43:10 [verbose]: 📺 OsuScreenStack#502(depth:1) suspended Loader#532 (waiting on Disclaimer#342)
2023-10-01 04:43:10 [verbose]: 📺 OsuScreenStack#502(depth:2) entered Disclaimer#342
2023-10-01 04:43:10 [verbose]: Found 0 scores which require reprocessing.
2023-10-01 04:43:10 [verbose]: Querying for scores that need total score conversion...
2023-10-01 04:43:10 [verbose]: Found 0 scores which require total score conversion.
2023-10-01 04:43:10 [verbose]: Finished background data store processing!
2023-10-01 04:43:11 [verbose]: Requesting catch-up from 7865284
2023-10-01 04:43:11 [verbose]: Requesting any changes since last known queue id 7865284
2023-10-01 04:43:11 [verbose]: Requesting catch-up from 7865289
2023-10-01 04:43:11 [verbose]: Requesting any changes since last known queue id 7865289
2023-10-01 04:43:11 [verbose]: Catch-up complete at 7865289
2023-10-01 04:43:12 [verbose]: Current channel changed to #osu
2023-10-01 04:43:16 [verbose]: 📺 OsuScreenStack#502(depth:2) suspended Disclaimer#342 (waiting on IntroTriangles#814)
2023-10-01 04:43:16 [verbose]: 📺 OsuScreenStack#502(depth:3) entered IntroTriangles#814
2023-10-01 04:43:16 [verbose]: 📺 BackgroundScreenStack#350(depth:1) loading BackgroundScreenDefault#509
2023-10-01 04:43:16 [verbose]: Game-wide working beatmap updated to cYsmix - triangles (peppy) [peppy]
2023-10-01 04:43:16 [verbose]: 🌅 Global background change queued
2023-10-01 04:43:16 [verbose]: 📺 BackgroundScreenStack#350(depth:1) entered BackgroundScreenDefault#509
2023-10-01 04:43:16 [verbose]: 🖍️ Shader LogoAnimation/LogoAnimation loaded from cache!
2023-10-01 04:43:16 [verbose]: decoupled ruleset transferred ("" -> "karaoke!")
2023-10-01 04:43:16 [verbose]: 🌅 Global background change queued
2023-10-01 04:43:17 [verbose]: 🌅 Global background loading
2023-10-01 04:43:19 [verbose]: 📺 OsuScreenStack#502(depth:3) suspended IntroTriangles#814 (waiting on MainMenu#741)
2023-10-01 04:43:19 [verbose]: 📺 OsuScreenStack#502(depth:4) entered MainMenu#741
2023-10-01 04:43:19 [verbose]: ⚠️ An unhandled error has occurred.
2023-10-01 04:43:19 [verbose]: 
2023-10-01 04:43:19 [verbose]: This error has been automatically reported to the devs.
2023-10-01 04:43:49 [verbose]: ButtonSystem's state changed from Initial to TopLevel
2023-10-01 04:43:50 [verbose]: Focus contention triggered by NotificationOverlay.
2023-10-01 04:43:50 [verbose]: Received beatmap updates 1 updates with last id 7865290
2023-10-01 04:43:59 [verbose]: ButtonSystem's state changed from TopLevel to Initial
2023-10-01 04:44:50 [verbose]: Received beatmap updates 1 updates with last id 7865291
2023-10-01 04:45:22 [verbose]: Game-wide working beatmap updated to Vickeblanka - Black Rover (TV Size) (Sotarks) [Akitoshi's Normal]
2023-10-01 04:45:22 [verbose]: 🌅 Global background change queued
2023-10-01 04:45:23 [verbose]: 🌅 Global background loading
2023-10-01 04:45:50 [verbose]: Received beatmap updates 1 updates with last id 7865292
2023-10-01 04:46:08 [verbose]: ButtonSystem's state changed from Initial to TopLevel
2023-10-01 04:46:09 [verbose]: ButtonSystem's state changed from TopLevel to Play
2023-10-01 04:46:11 [verbose]: 📺 OsuScreenStack#502(depth:4) suspended MainMenu#741 (waiting on PlaySongSelect#306)
2023-10-01 04:46:11 [verbose]: ButtonSystem's state changed from Play to EnteringMode
2023-10-01 04:46:11 [verbose]: 📺 OsuScreenStack#502(depth:5) entered PlaySongSelect#306
2023-10-01 04:46:11 [verbose]: 📺 BackgroundScreenStack#350(depth:2) loading BackgroundScreenBeatmap#262
2023-10-01 04:46:11 [verbose]: 📺 BackgroundScreenStack#350(depth:2) suspended BackgroundScreenDefault#509 (waiting on BackgroundScreenBeatmap#262)
2023-10-01 04:46:11 [verbose]: 📺 BackgroundScreenStack#350(depth:2) entered BackgroundScreenBeatmap#262
2023-10-01 04:46:11 [verbose]: Song select updating selection with beatmap:6ee8d52b-05b5-4dab-a193-0bdca044e161 ruleset:karaoke!
2023-10-01 04:46:12 [verbose]: Focus contention triggered by SettingsOverlay.
2023-10-01 04:46:14 [error]: Failed to load ruleset settings for karaoke!. Please check for an update from the developer.
2023-10-01 04:46:14 [verbose]: ⚠️ Failed to load ruleset settings for karaoke!. Please check for an update from the developer.
2023-10-01 04:46:16 [verbose]: Scrolling to AudioSection..
2023-10-01 04:46:16 [verbose]: Reattempting scroll to AudioSection due to change in position
2023-10-01 04:46:16 [verbose]: Scrolling to AudioSection..
2023-10-01 04:46:17 [verbose]: Finished scrolling to AudioSection!
2023-10-01 04:46:17 [verbose]: Scrolling to RulesetSection..
2023-10-01 04:46:17 [verbose]: Finished scrolling to RulesetSection!
2023-10-01 04:46:28 [verbose]: Focus contention triggered by NotificationOverlay.
2023-10-01 04:46:29 [verbose]: 📺 BackgroundScreenStack#350(depth:1) exit from BackgroundScreenBeatmap#262
2023-10-01 04:46:29 [verbose]: 📺 BackgroundScreenStack#350(depth:1) resume to BackgroundScreenDefault#509
2023-10-01 04:46:29 [verbose]: 📺 OsuScreenStack#502(depth:4) exit from PlaySongSelect#306
2023-10-01 04:46:29 [verbose]: 📺 OsuScreenStack#502(depth:4) resume to MainMenu#741
2023-10-01 04:46:29 [verbose]: decoupled ruleset transferred ("" -> "karaoke!")
2023-10-01 04:46:29 [verbose]: 🌅 Global background change queued
2023-10-01 04:46:29 [verbose]: ButtonSystem's state changed from EnteringMode to TopLevel
2023-10-01 04:46:29 [verbose]: 🌅 Global background loading
2023-10-01 04:46:30 [verbose]: ButtonSystem's state changed from TopLevel to Exit
2023-10-01 04:46:30 [verbose]: 📺 OsuScreenStack#502(depth:3) exit from MainMenu#741
2023-10-01 04:46:30 [verbose]: 📺 OsuScreenStack#502(depth:3) resume to IntroTriangles#814
2023-10-01 04:46:33 [verbose]: 📺 BackgroundScreenStack#350(depth:0) exit from BackgroundScreenDefault#509
2023-10-01 04:46:33 [verbose]: 📺 BackgroundScreenStack#350(depth:0) resume to [empty]
2023-10-01 04:46:33 [verbose]: 📺 OsuScreenStack#502(depth:2) exit from IntroTriangles#814
2023-10-01 04:46:33 [verbose]: 📺 OsuScreenStack#502(depth:2) resume to Disclaimer#342
2023-10-01 04:46:33 [verbose]: 📺 OsuScreenStack#502(depth:1) exit from Disclaimer#342
2023-10-01 04:46:33 [verbose]: 📺 OsuScreenStack#502(depth:1) resume to Loader#532
2023-10-01 04:46:33 [verbose]: 📺 OsuScreenStack#502(depth:0) exit from Loader#532
2023-10-01 04:46:33 [verbose]: 📺 OsuScreenStack#502(depth:0) resume to [empty]
2023-10-01 04:46:33 [verbose]: Host execution state changed to Stopping
2023-10-01 04:46:33 [verbose]: Host execution state changed to Stopped
```

I basically just straight up downloading karaoke release .dll and put it on C:\Users\name\AppData\Roaming\osu\rulesets. But It will just show up 2 error of couldn't loading ruleset of karaoke. I had already remove all the other .dll in the file and reinstall osu!lazer already. Not fixed.

I have no idea what the error is about by the way.

**Comment by andy840119** [comment]
Try to download the .zip file and un-zip into the ruleset folder instead?

![image](https://github.com/karaoke-dev/karaoke/assets/9100368/34e655b7-fa77-46dc-8c60-3cd0027c265c)
Like this.

**Comment by Sewlell** [comment]
```
2023-10-01 05:36:19 [verbose]: ----------------------------------------------------------
2023-10-01 05:36:19 [verbose]: runtime Log for user (LogLevel: Verbose)
2023-10-01 05:36:19 [verbose]: Running osu 2023.924.1.0 on .NET 6.0.22
2023-10-01 05:36:19 [verbose]: Environment: Windows (Microsoft Windows NT 10.0.22621.0), 8 cores
2023-10-01 05:36:19 [verbose]: ----------------------------------------------------------
2023-10-01 05:36:19 [verbose]: 🖼️ Configuration renderer choice: Automatic
2023-10-01 05:36:19 [verbose]: 🖼️ Renderer fallback order: [ Direct3D 11, OpenGL (Legacy), OpenGL, Vulkan ]
2023-10-01 05:36:19 [verbose]: 🖼️ Initialising "Veldrid" renderer with "Direct3D11" surface
2023-10-01 05:36:19 [verbose]: Updated display mode to desktop resolution: 1920x1080@60, SDL_PIXELFORMAT_RGB888
2023-10-01 05:36:20 [verbose]: Direct3D 11 Initialized
2023-10-01 05:36:20 [verbose]: Direct3D 11 Feature Level:           11.0
2023-10-01 05:36:20 [verbose]: Direct3D 11 Adapter:                 NVIDIA GeForce MX130
2023-10-01 05:36:20 [verbose]: Direct3D 11 Dedicated Video Memory:  2008 MB
2023-10-01 05:36:20 [verbose]: Direct3D 11 Dedicated System Memory: 0 MB
2023-10-01 05:36:20 [verbose]: Direct3D 11 Shared System Memory:    6045 MB
2023-10-01 05:36:20 [verbose]: UseStructuredBuffers: True
2023-10-01 05:36:20 [verbose]: 🖼️ Renderer initialised!
2023-10-01 05:36:20 [verbose]: Host execution state changed to Running
2023-10-01 05:36:20 [verbose]: Execution mode changed to MultiThreaded
2023-10-01 05:36:20 [verbose]: Beginning realm file store cleanup
2023-10-01 05:36:20 [verbose]: Finished realm file store cleanup (0 of 20808 deleted)
**2023-10-01 05:36:20 [error]: Could not load ruleset "Resources". Please check for an update from the developer.**
2023-10-01 05:36:20 [verbose]: Ruleset load failed: System.InvalidOperationException: Sequence contains no matching element
2023-10-01 05:36:20 [verbose]: at System.Linq.ThrowHelper.ThrowNoMatchException()
2023-10-01 05:36:20 [verbose]: at System.Linq.Enumerable.First[TSource](IEnumerable`1 source, Func`2 predicate)
2023-10-01 05:36:20 [verbose]: at osu.Game.Rulesets.RulesetStore.addRuleset(Assembly assembly)
2023-10-01 05:36:20 [verbose]: Opened realm "C:\Users\user\AppData\Roaming\osu\client.realm" at version 34
2023-10-01 05:36:20 [verbose]: 🔈 BASS initialised
2023-10-01 05:36:20 [verbose]: BASS version:           2.4.16.7
2023-10-01 05:36:20 [verbose]: BASS FX version:        2.4.12.6
2023-10-01 05:36:20 [verbose]: BASS MIX version:       2.4.11.1
2023-10-01 05:36:20 [verbose]: Device:                 Default
2023-10-01 05:36:20 [verbose]: Driver:
2023-10-01 05:36:20 [verbose]: Update period:          5 ms
2023-10-01 05:36:20 [verbose]: Device buffer length:   10 ms
2023-10-01 05:36:20 [verbose]: Playback buffer length: 100 ms
2023-10-01 05:36:21 [verbose]: 🖍️ Shader Texture2D/Texture loaded from cache!
2023-10-01 05:36:21 [verbose]: 🖍️ Shader Texture2D/CircularProgress loaded from cache!
2023-10-01 05:36:21 [verbose]: Updated display mode to fullscreen resolution: 1920x1080@60, SDL_PIXELFORMAT_RGB888
2023-10-01 05:36:21 [verbose]: Loading FPSCounter...
**2023-10-01 05:36:22 [error]: An unhandled error has occurred.
2023-10-01 05:36:22 [error]: System.Globalization.CultureNotFoundException: Culture is not supported. (Parameter 'name')
2023-10-01 05:36:22 [error]: 中文（简体） is an invalid culture identifier.
2023-10-01 05:36:22 [error]: at System.Globalization.CultureInfo..ctor(String name, Boolean useUserOverride)
2023-10-01 05:36:22 [error]: at System.Globalization.CultureInfo..ctor(String name)
2023-10-01 05:36:22 [error]: at osu.Game.Rulesets.Karaoke.Utils.CultureInfoUtils.CreateLoadCultureInfoByCode(String code)
2023-10-01 05:36:22 [error]: at osu.Game.Rulesets.Karaoke.Bindables.BindableCultureInfo.Parse(Object input)
2023-10-01 05:36:22 [error]: at osu.Game.Rulesets.Configuration.RulesetConfigManager`1.AddBindable[TBindable](TLookup lookup, Bindable`1 bindable)
2023-10-01 05:36:22 [error]: at osu.Game.Rulesets.Karaoke.Configuration.KaraokeRulesetConfigManager.AddBindable[TBindable](KaraokeRulesetSetting lookup, Bindable`1 bindable)
2023-10-01 05:36:22 [error]: at osu.Framework.Configuration.ConfigManager`1.set[TValue](TLookup lookup, TValue value)
2023-10-01 05:36:22 [error]: at osu.Framework.Configuration.ConfigManager`1.SetDefault[TValue](TLookup lookup, TValue value)
2023-10-01 05:36:22 [error]: at osu.Game.Rulesets.Karaoke.Configuration.KaraokeRulesetConfigManager.InitialiseDefaults()
2023-10-01 05:36:22 [error]: at osu.Game.Rulesets.Configuration.RulesetConfigManager`1..ctor(SettingsStore store, RulesetInfo ruleset, Nullable`1 variant)
2023-10-01 05:36:22 [error]: at osu.Game.Rulesets.Karaoke.Configuration.KaraokeRulesetConfigManager..ctor(SettingsStore settings, RulesetInfo ruleset, Nullable`1 variant)
2023-10-01 05:36:22 [error]: at osu.Game.Rulesets.Karaoke.KaraokeRuleset.CreateConfig(SettingsStore settings)
2023-10-01 05:36:22 [error]: at osu.Game.Rulesets.RulesetConfigCache.LoadComplete()
2023-10-01 05:36:22 [error]: at osu.Framework.Graphics.Drawable.loadComplete()
2023-10-01 05:36:22 [error]: at osu.Framework.Graphics.Drawable.UpdateSubTree()
2023-10-01 05:36:22 [error]: at osu.Framework.Graphics.Containers.CompositeDrawable.UpdateSubTree()
2023-10-01 05:36:22 [error]: at osu.Framework.Graphics.Containers.CompositeDrawable.UpdateSubTree()
2023-10-01 05:36:22 [error]: at osu.Framework.Graphics.Containers.CompositeDrawable.UpdateSubTree()
2023-10-01 05:36:22 [error]: at osu.Framework.Graphics.Containers.CompositeDrawable.UpdateSubTree()
2023-10-01 05:36:22 [error]: at osu.Framework.Graphics.Containers.CompositeDrawable.UpdateSubTree()
2023-10-01 05:36:22 [error]: at osu.Framework.Graphics.Containers.CompositeDrawable.UpdateSubTree()
2023-10-01 05:36:22 [error]: at osu.Framework.Platform.GameHost.UpdateFrame()
2023-10-01 05:36:22 [error]: at osu.Framework.Threading.GameThread.processFrame()**
2023-10-01 05:36:22 [verbose]: Unhandled exception has been allowed with 0 more allowable exceptions .
2023-10-01 05:36:22 [verbose]: Loaded FPSCounter!
2023-10-01 05:36:22 [verbose]: Loading VersionManager...
2023-10-01 05:36:22 [verbose]: Loaded VersionManager!
2023-10-01 05:36:22 [verbose]: Loading OsuLogo...
2023-10-01 05:36:22 [verbose]: 🖍️ Shader Texture2D/TriangleBorder loaded from cache!
2023-10-01 05:36:22 [verbose]: Loaded OsuLogo!
2023-10-01 05:36:22 [verbose]: Loading Toolbar...
2023-10-01 05:36:22 [verbose]: 📺 OsuScreenStack#502(depth:1) loading Loader#532
2023-10-01 05:36:22 [verbose]: 📺 OsuScreenStack#502(depth:1) entered Loader#532
2023-10-01 05:36:22 [verbose]: 🖍️ Shader Texture2D/Blur loaded from cache!
2023-10-01 05:36:22 [verbose]: 🖍️ Shader CursorTrail/Texture loaded from cache!
2023-10-01 05:36:22 [verbose]: 🖍️ Shader Texture3D/Texture loaded from cache!
2023-10-01 05:36:23 [verbose]: Loaded Toolbar!
2023-10-01 05:36:23 [verbose]: Loading VolumeOverlay...
2023-10-01 05:36:23 [verbose]: Loaded VolumeOverlay!
2023-10-01 05:36:23 [verbose]: Loading OnScreenDisplay...
2023-10-01 05:36:23 [verbose]: Loaded OnScreenDisplay!
2023-10-01 05:36:23 [verbose]: Loading NotificationOverlay...
2023-10-01 05:36:23 [verbose]: Loaded NotificationOverlay!
2023-10-01 05:36:23 [verbose]: Loading LegacyImportManager...
2023-10-01 05:36:23 [verbose]: Loaded LegacyImportManager!
2023-10-01 05:36:23 [verbose]: Loading ScreenshotManager...
2023-10-01 05:36:23 [verbose]: Loaded ScreenshotManager!
2023-10-01 05:36:23 [verbose]: Loading SquirrelUpdateManager...
2023-10-01 05:36:23 [verbose]: Loaded SquirrelUpdateManager!
2023-10-01 05:36:23 [verbose]: Loading FirstRunSetupOverlay...
2023-10-01 05:36:23 [verbose]: Loaded FirstRunSetupOverlay!
2023-10-01 05:36:23 [verbose]: Loading ManageCollectionsDialog...
2023-10-01 05:36:23 [verbose]: Loaded ManageCollectionsDialog!
2023-10-01 05:36:23 [verbose]: Loading BeatmapListingOverlay...
2023-10-01 05:36:23 [verbose]: Loaded BeatmapListingOverlay!
2023-10-01 05:36:23 [verbose]: Loading DashboardOverlay...
2023-10-01 05:36:23 [verbose]: Loaded DashboardOverlay!
2023-10-01 05:36:23 [verbose]: Loading NewsOverlay...
2023-10-01 05:36:23 [verbose]: Loaded NewsOverlay!
2023-10-01 05:36:23 [verbose]: Loading RankingsOverlay...
2023-10-01 05:36:23 [verbose]: Loaded RankingsOverlay!
2023-10-01 05:36:23 [verbose]: Loading ChannelManager...
2023-10-01 05:36:23 [verbose]: Loaded ChannelManager!
2023-10-01 05:36:23 [verbose]: Loading ChatOverlay...
2023-10-01 05:36:23 [verbose]: Loaded ChatOverlay!
2023-10-01 05:36:23 [verbose]: Loading MessageNotifier...
2023-10-01 05:36:23 [verbose]: Loaded MessageNotifier!
2023-10-01 05:36:23 [verbose]: Loading SettingsOverlay...
2023-10-01 05:36:23 [verbose]: Loaded SettingsOverlay!
2023-10-01 05:36:23 [verbose]: Loading ChangelogOverlay...
2023-10-01 05:36:23 [verbose]: Loaded ChangelogOverlay!
2023-10-01 05:36:23 [verbose]: Loading UserProfileOverlay...
2023-10-01 05:36:23 [verbose]: Loaded UserProfileOverlay!
2023-10-01 05:36:23 [verbose]: Loading BeatmapSetOverlay...
2023-10-01 05:36:23 [verbose]: Loaded BeatmapSetOverlay!
2023-10-01 05:36:23 [verbose]: Loading WikiOverlay...
2023-10-01 05:36:23 [verbose]: Loaded WikiOverlay!
2023-10-01 05:36:23 [verbose]: Loading SkinEditorOverlay...
2023-10-01 05:36:23 [verbose]: Loaded SkinEditorOverlay!
2023-10-01 05:36:23 [verbose]: Loading LoginOverlay...
2023-10-01 05:36:23 [verbose]: Loaded LoginOverlay!
2023-10-01 05:36:23 [verbose]: Loading NowPlayingOverlay...
2023-10-01 05:36:24 [verbose]: Loaded NowPlayingOverlay!
2023-10-01 05:36:24 [verbose]: Loading AccountCreationOverlay...
2023-10-01 05:36:24 [verbose]: 📺 ScreenStack#124(depth:1) scheduling push ScreenWelcome#263
2023-10-01 05:36:24 [verbose]: Loaded AccountCreationOverlay!
2023-10-01 05:36:24 [verbose]: Loading DialogOverlay...
2023-10-01 05:36:24 [verbose]: Loaded DialogOverlay!
2023-10-01 05:36:24 [verbose]: Loading HighPerformanceSession...
2023-10-01 05:36:24 [verbose]: Requesting catch-up from 7865353
2023-10-01 05:36:24 [verbose]: Requesting any changes since last known queue id 7865353
2023-10-01 05:36:24 [verbose]: Loaded HighPerformanceSession!
2023-10-01 05:36:24 [verbose]: Loading BackgroundDataStoreProcessor...
2023-10-01 05:36:24 [verbose]: Loaded BackgroundDataStoreProcessor!
2023-10-01 05:36:24 [verbose]: 📺 OsuScreenStack#502(depth:1) suspended Loader#532 (waiting on Disclaimer#342)
2023-10-01 05:36:24 [verbose]: 📺 OsuScreenStack#502(depth:2) entered Disclaimer#342
2023-10-01 05:36:24 [verbose]: Beginning background data store processing..
2023-10-01 05:36:24 [verbose]: Querying for beatmap sets to reprocess...
2023-10-01 05:36:24 [verbose]: Found 0 beatmap sets which require reprocessing.
2023-10-01 05:36:24 [verbose]: Querying for scores to reprocess...
2023-10-01 05:36:24 [verbose]: Found 0 scores which require reprocessing.
2023-10-01 05:36:24 [verbose]: Querying for scores that need total score conversion...
2023-10-01 05:36:24 [verbose]: Found 0 scores which require total score conversion.
2023-10-01 05:36:24 [verbose]: Finished background data store processing!
2023-10-01 05:36:24 [verbose]: Requesting catch-up from 7865355
2023-10-01 05:36:24 [verbose]: Requesting any changes since last known queue id 7865355
2023-10-01 05:36:24 [verbose]: Catch-up complete at 7865355
2023-10-01 05:36:25 [verbose]: Current channel changed to #osu
2023-10-01 05:36:30 [verbose]: 📺 OsuScreenStack#502(depth:2) suspended Disclaimer#342 (waiting on IntroTriangles#490)
2023-10-01 05:36:30 [verbose]: 📺 OsuScreenStack#502(depth:3) entered IntroTriangles#490
2023-10-01 05:36:30 [verbose]: 📺 BackgroundScreenStack#549(depth:1) loading BackgroundScreenDefault#282
2023-10-01 05:36:30 [verbose]: Game-wide working beatmap updated to cYsmix - triangles (peppy) [peppy]
2023-10-01 05:36:30 [verbose]: 🌅 Global background change queued
2023-10-01 05:36:30 [verbose]: 📺 BackgroundScreenStack#549(depth:1) entered BackgroundScreenDefault#282
2023-10-01 05:36:30 [verbose]: decoupled ruleset transferred ("" -> "osu!")
2023-10-01 05:36:30 [verbose]: 🖍️ Shader LogoAnimation/LogoAnimation loaded from cache!
2023-10-01 05:36:30 [verbose]: 🌅 Global background loading
2023-10-01 05:36:31 [verbose]: 🌅 Global background change queued
2023-10-01 05:36:31 [verbose]: 🌅 Global background loading
2023-10-01 05:36:33 [verbose]: 📺 OsuScreenStack#502(depth:3) suspended IntroTriangles#490 (waiting on MainMenu#426)
2023-10-01 05:36:33 [verbose]: 📺 OsuScreenStack#502(depth:4) entered MainMenu#426
2023-10-01 05:36:33 [verbose]: ⚠️ Could not load ruleset "Resources". Please check for an update from the developer.
2023-10-01 05:36:33 [verbose]: ⚠️ An unhandled error has occurred.
2023-10-01 05:36:33 [verbose]: 
2023-10-01 05:36:33 [verbose]: This error has been automatically reported to the devs.
2023-10-01 05:36:44 [verbose]: ButtonSystem's state changed from Initial to TopLevel
2023-10-01 05:36:49 [error]: Failed to load ruleset settings for karaoke!. Please check for an update from the developer.
2023-10-01 05:36:49 [verbose]: ⚠️ Failed to load ruleset settings for karaoke!. Please check for an update from the developer.
2023-10-01 05:36:50 [verbose]: Scrolling to AudioSection..
2023-10-01 05:36:50 [verbose]: Reattempting scroll to AudioSection due to change in position
2023-10-01 05:36:50 [verbose]: Scrolling to AudioSection..
2023-10-01 05:36:51 [verbose]: Scrolling to AudioSection..
2023-10-01 05:36:51 [verbose]: Scrolling to RulesetSection..
2023-10-01 05:36:52 [verbose]: Scroll operation interrupted by user scroll
2023-10-01 05:36:56 [verbose]: ButtonSystem's state changed from TopLevel to Play
2023-10-01 05:36:56 [verbose]: 📺 OsuScreenStack#502(depth:4) suspended MainMenu#426 (waiting on PlaySongSelect#560)
2023-10-01 05:36:56 [verbose]: ButtonSystem's state changed from Play to EnteringMode
2023-10-01 05:36:56 [verbose]: 📺 OsuScreenStack#502(depth:5) entered PlaySongSelect#560
2023-10-01 05:36:56 [verbose]: 📺 BackgroundScreenStack#549(depth:2) loading BackgroundScreenBeatmap#298
2023-10-01 05:36:56 [verbose]: 📺 BackgroundScreenStack#549(depth:2) suspended BackgroundScreenDefault#282 (waiting on BackgroundScreenBeatmap#298)
2023-10-01 05:36:56 [verbose]: 📺 BackgroundScreenStack#549(depth:2) entered BackgroundScreenBeatmap#298
2023-10-01 05:36:56 [verbose]: Song select updating selection with beatmap:250d4650-8271-4b71-94bd-b5ea235443ff ruleset:osu
2023-10-01 05:36:56 [verbose]: Song select changing beatmap from "cYsmix - triangles (peppy) [peppy]" to "SakiZ - osu!memories (DeRandom Otaku) [Happy Memories]"
2023-10-01 05:36:56 [verbose]: Game-wide working beatmap updated to SakiZ - osu!memories (DeRandom Otaku) [Happy Memories]
2023-10-01 05:36:56 [verbose]: Song select working beatmap updated to SakiZ - osu!memories (DeRandom Otaku) [Happy Memories]
2023-10-01 05:36:57 [verbose]: Song select decided to ensurePlayingSelected
2023-10-01 05:36:57 [verbose]: 🌅 Global background change queued
2023-10-01 05:36:57 [verbose]: Song select updating selection with beatmap:250d4650-8271-4b71-94bd-b5ea235443ff ruleset:karaoke!
2023-10-01 05:36:57 [verbose]: decoupled ruleset transferred ("osu!" -> "karaoke!")
2023-10-01 05:36:59 [verbose]: 📺 OsuScreenStack#502(depth:5) suspended PlaySongSelect#560 (waiting on PlayerLoader#533)
2023-10-01 05:36:59 [verbose]: 📺 OsuScreenStack#502(depth:6) loading PlayerLoader#533
2023-10-01 05:36:59 [verbose]: 📺 OsuScreenStack#502(depth:6) entered PlayerLoader#533
2023-10-01 05:37:03 [verbose]: Focus contention triggered by NotificationOverlay.
2023-10-01 05:37:04 [verbose]: Score submission token retrieval failed (Request could not be constructed.)
2023-10-01 05:37:04 [verbose]: Focus contention triggered by NotificationOverlay.
2023-10-01 05:37:10 [verbose]: 📺 OsuScreenStack#502(depth:6) suspended PlayerLoader#533 (waiting on SoloPlayer#260)
2023-10-01 05:37:10 [verbose]: 📺 OsuScreenStack#502(depth:7) entered SoloPlayer#260
2023-10-01 05:37:10 [verbose]: GameplayClockContainer seeking to -1555
2023-10-01 05:37:21 [verbose]: 📺 OsuScreenStack#502(depth:6) exit from SoloPlayer#260
2023-10-01 05:37:21 [verbose]: 📺 OsuScreenStack#502(depth:6) resume to PlayerLoader#533
2023-10-01 05:37:21 [verbose]: 📺 OsuScreenStack#502(depth:5) exit from PlayerLoader#533
2023-10-01 05:37:21 [verbose]: 📺 OsuScreenStack#502(depth:5) resume to PlaySongSelect#560
2023-10-01 05:37:21 [verbose]: Song select working beatmap updated to SakiZ - osu!memories (DeRandom Otaku) [Happy Memories]
2023-10-01 05:37:21 [verbose]: Song select updating selection with beatmap:250d4650-8271-4b71-94bd-b5ea235443ff ruleset:karaoke!
2023-10-01 05:37:21 [verbose]: Focus contention triggered by NotificationOverlay.
2023-10-01 05:37:47 [verbose]: Received beatmap updates 1 updates with last id 7865356
2023-10-01 05:37:52 [verbose]: Received beatmap updates 1 updates with last id 7865357
2023-10-01 05:37:57 [verbose]: Received beatmap updates 2 updates with last id 7865359
```

still a no, and now with new error, Could not load ruleset "Resources", other two still remained

**Comment by andy840119** [comment]
Guess found the root cause.
Has initialize error while making the ruleset config.

你用的是簡體中文的 windows 10 對吧?

**Comment by Sewlell** [comment]
我是美国英文的Windows 11啊
Mine is American English Windows 11.

我是回想到因为某种程式只接受日本用户，那时候我就把国家/地区改成日本。我现在改回马来西亚了好像也没有变化。
I remembered that due to some app only accepted Japanese users ( If i wont wrong ), I change the Country/Region setting to Japan. Now I revert it to Malaysia, it doesn't seem to make any changes thought.

( Dual language because eh )

![Screenshot (255)](https://github.com/karaoke-dev/karaoke/assets/93970226/259f5201-dff7-4110-9838-40e38a0bcabc)

**Comment by andy840119** [comment]
![image](https://github.com/karaoke-dev/karaoke/assets/9100368/4b4f89e3-b321-413d-8a88-41af05c5ea24)
Did you change the prefer language before?

Create a PR(#2142) for fixing it, trying to download the latest update again(2023.1001.0) and this issue should be fixed.

Tell me if this issue is fixed 🙆🏼‍♂️

**Comment by Sewlell** [comment]
Odd... I hadn't. My memory is like I just slap this ruleset's .dll quite couple of times in 2021 and it just don't work. I am "feared" to report that bug to dev for some reason.

Oh btw, the Simplified Chinese is still unable to parse. But that's fine since my Chinese is F :cry: . Anyway thank for helping, I could enter the karaoke ruleset setting and editor now.

![Screenshot (256)](https://github.com/karaoke-dev/karaoke/assets/93970226/007e9605-4a99-4ba2-a87c-ebee39cbc1ef)

P.S. Busy with schoolwork.

**Comment by andy840119** [comment]
It's OK to report the issue at any time.
If the issue is not clear enough, i will ask for more detail 👍🏼

**Comment by DFlow69** [comment]
Huh? I read trough the issue but I didn't understand how to fix "Could not load 'Resources'"? Please help.

**Comment by andy840119** [comment]
@DFlow69 because lazer will load any resource that prefix with osu.game.ruleset.xxx

karaoke ruleset has 2 dlls.
one is main game, another one is resource (osu.game.ruleset.karaoke.resource).
now it's not possible not to load this dll without modify the lazer.

And the error message will not breaking anything, so just ignore it now.

**Comment by DFlow69** [comment]
@andy840119 if this isn't an issue then why every time when I open the karaoke editor it keeps crashing (I thought it was because resources wasn't loading)? Also if I downgrade my lazer version will it work and if not then how exactly do I make maps (manually seems hard cuz I tried but it just doesn't detect hit objects (copied from demo))?

**Comment by andy840119** [comment]
> I thought it was because resources wasn't loading

resource should be loaded after lazer is open i guess.

> when I open the karaoke editor it keeps crashing 

Editor sometimes crash due to lazer changes their api or other issue, could you provide the beatmap you are using or opened?
Also, could you provide the log after goes to the editor? Open a new issue with the log.
